circuit-construction/tests/example_proof.rs

```diff
@@ -64,23 +64,20 @@ const PUBLIC_INPUT_LENGTH: usize = 3;
 #[test]
 fn test_example_circuit() {
     // create SRS
+    let proof_system_constants = fp_constants();
     let srs = {
-        let mut srs = SRS::<Affine>::create(1 << 7); // 2^7 = 128
+        let mut srs = SRS::<Affine>::create(1 << 7, proof_system_constants.poseidon.clone()); // 2^7 = 128
```

Comment:
I'm wondering if we really have to pass that as argument. In the same vein we generate `endo_q` and `endo_r`, couldn't we generate this automatically in `SRS::create`?